src/main/java/net/neoforged/neoforge/event/entity/living/FinalizeSpawnEvent.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.event.entity.living;
+
+import com.mojang.datafixers.util.Either;
+import net.minecraft.server.level.WorldGenRegion;
+import net.minecraft.world.DifficultyInstance;
+import net.minecraft.world.entity.Entity;
+import net.minecraft.world.entity.Mob;
+import net.minecraft.world.entity.MobSpawnType;
+import net.minecraft.world.entity.SpawnGroupData;
+import net.minecraft.world.entity.vehicle.MinecartSpawner;
+import net.minecraft.world.level.ServerLevelAccessor;
+import net.minecraft.world.level.block.entity.BlockEntity;
+import net.minecraft.world.level.block.entity.SpawnerBlockEntity;
+import net.minecraft.world.level.block.entity.TrialSpawnerBlockEntity;
+import net.neoforged.bus.api.ICancellableEvent;
+import net.neoforged.neoforge.common.NeoForge;
+import net.neoforged.neoforge.common.NeoForgeEventHandler;
+import net.neoforged.neoforge.event.EventHooks;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * This event is fired before {@link Mob#finalizeSpawn} is called.<br>
```

Comment:
`<br>` is bad practice. Use paragraphs where it makes sense, let the renderer take care of line breaks within paragraphs.

Reply:
This is just what was already there; I can go update it, but the event is unchanged aside from the `spawner` field